.github/workflows/pr-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    name: Unit
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17.6
```

Comment:
remove `^` - it can fetch go 1.19 which is not currently supported by the node

Reply:
changed